src/main/jbake/content/download.adoc

```diff
@@ -12,8 +12,8 @@ You **must** link:https://www.apache.org/info/verification.html[verify] the inte
 == [[tomee-9.0]]Apache TomEE 9.0
 
 - Jakarta EE 9.1
-- MicroProfile 2.0
```

Comment:
While this change is true for the current M8-SNAPSHOT, it isn't for the latest M7 release, which bundles the old MP versions. In addition, it is compiled against Java 11 as we used byte code transformation of TomEE 8 to create M7. 

I would suggest to drop this particular change until we have an M8 or final release.